diff --git a/common.sh b/common.sh
--- a/common.sh
+++ b/common.sh
@@ -16,7 +16,7 @@ export TERMUX_PACKAGE_NAME
 
 # Package name of the intended application
-COTG_PACKAGE_NAME="com.itsaky.androidide"
+COTG_PACKAGE_NAME="com.jepongdevxyz.idebuild"
 export COTG_PACKAGE_NAME
 
 # Path to the public key
diff --git a/packages.sh b/packages.sh
--- a/packages.sh
+++ b/packages.sh
@@ -42,6 +42,7 @@ COTG_PACKAGES__BASE=(
     "nano"
     "net-tools"
     "openjdk-21"
+    "aapt"
     "patch"
     "unzip"
     "zip"
